atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/transaction/impl/TestTransactionManagerImpl.java

```diff
@@ -48,10 +50,30 @@
     private final Map<TableReference, ConflictHandler> conflictHandlerOverrides = new HashMap<>();
     private Optional<Long> unreadableTs = Optional.empty();
 
+
+    /**
+     * Utility constructor for when we know we have an InMemoryTimestampService.
+     */
+    public TestTransactionManagerImpl(MetricsManager metricsManager,
```

Comment:
I feel less strongly about this one, since it's test code, but maybe we don't need this constructor either, given that we already have several and this one isn't meaningfully different.